### PR TITLE
Fix PR08, RT02, RT03, and SA01 on pandas.Index.fillna

### PR DESCRIPTION
**Pull request by RafifEL**
- [x] closes #https://github.com/pandanistas/pandanistas_sprint_ui2020/issues/5
- [ ] tests added / passed
- [x] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by MomIsBestFriend** [comment]
Why was all of those files in c4099b5 was removed?

**Comment by WillAyd** [comment]
@RafifEL looks like something went awry with the git workflow here. Can you `git checkout master FILENAME` for all the files you didn't update the docstring in and re-push?

**Comment by RafifEL** [comment]
I'm sorry I'm new to this stuff. is there any way for me to go back to 1dbea46 commit?

**Comment by RafifEL** [comment]
> @RafifEL looks like something went awry with the git workflow here. Can you `git checkout master FILENAME` for all the files you didn't update the docstring in and re-push?

I'm sorry, I'm new to this stuff, is there anyway for me to hard reset to 1dbea46 commit?

**Comment by WillAyd** [comment]
Sure! From your branch try:

```sh
git reset --hard 1dbea46
git push -f origin rafif-feature
```

**Comment by RafifEL** [comment]
> Why was all of those files in [c4099b5](https://github.com/pandas-dev/pandas/commit/c4099b526ec0e1add6577f5d07d753860fce9b38) was removed?

I'm Sorry, i was trying to see the HTML of the documentation by using "python doc/make.py html". It got an Error, so i delete the reference folder on this branch

File: pandas/core/indexes/base.py [hunk 1]
```diff
@@ -2126,13 +2126,19 @@ def fillna(self, value=None, downcast=None):
...
         downcast : dict, default is None
-            a dict of item->dtype of what to downcast if possible,
+            A dict of item->dtype of what to downcast if possible,
             or the string 'infer' which will try to downcast to an appropriate
             equal type (e.g. float64 to int64 if possible).
 
         Returns
         -------
-        filled : Index
+        filled
```
Comment (WillAyd) [inline pandas/core/indexes/base.py]:
```suggestion
        Index
```

Can just specify return type; don't need named variable in Returns section

File: pandas/core/indexes/base.py [hunk 2]
```diff
@@ -2126,13 +2126,19 @@ def fillna(self, value=None, downcast=None):
...
+            Index.
```
Comment (WillAyd) [inline pandas/core/indexes/base.py]: Let's remove this line altogether

**Review by WillAyd — CHANGES_REQUESTED** [review]
Nice job getting back to a clean state. Just some minor fixups otherwise should be good to go here

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @RafifEL

**Pull request merged**